.github/workflows/build-plugin-zip.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Gutenberg Plugin Zip
+
+on:
+  pull_request:
+    paths-ignore:
+    - '**.md'
+  push:
+    branches: [master]
```

Comment:
I assume this is a different yaml array syntax, but I expected a single-item list like:

```yaml
  branches:
    - master
```

Reply:
Good point! FWIW, this was copy-pasted from the existing unit-test GH action, and I think that most of the other actions also have it like this, but we can change it here (and maybe file a PR to change it for the existing actions).